src/main/java/com/artipie/api/StorageAliasesRest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.api;
+
+import com.artipie.asto.Key;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.settings.Layout;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.openapi.RouterBuilder;
+import org.eclipse.jetty.http.HttpStatus;
+
+/**
+ * Rest API methods to manage storage aliases.
+ * @since 0.27
+ */
+public final class StorageAliasesRest extends BaseRest {
+
+    /**
+     * Artipie settings storage.
+     */
+    private final BlockingStorage asto;
+
+    /**
+     * Artipie layout.
+     */
+    private final String layout;
+
+    /**
+     * Ctor.
+     * @param asto Artipie settings storage
+     * @param layout Artipie layout
+     */
+    public StorageAliasesRest(final BlockingStorage asto, final String layout) {
+        this.asto = asto;
+        this.layout = layout;
+    }
+
+    @Override
+    public void init(final RouterBuilder rtrb) {
+        if (new Layout.Flat().toString().equals(this.layout)) {
+            rtrb.operation("getAliases")
+                .handler(this::getAliases)
+                .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+            rtrb.operation("getRepoAliases")
+                .handler(this::getRepoAliases)
+                .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+        }
+    }
+
+    /**
+     * Get repository aliases.
+     * @param context Routing context
+     */
+    private void getAliases(final RoutingContext context) {
+        context.response().setStatusCode(HttpStatus.OK_200).end(
+            JsonArray.of(
+                new ManageStorageAliases(this.asto).list().stream()
+                    .map(item -> new JsonObject(item.toString())).toArray()
+            ).encode()
+        );
```

Comment:
Here we have some convertions:
1. ManageStorageAliases(this.asto).list() 
    that returns Collection<javax.json.JsonObject>
2. .map(item -> new JsonObject(item.toString())
   convert each javax.json.JsonObject to String
   and each String to vertx.JsonObject
3. Creating vertx array: JsonArray.of
4. encode vertx JsonArray to String

I seam we have extra encoding from one Json to another Json.
So it can be simplified by adding all javax.json.JsonObject to one javax.json.JsonArray and after that convert it to String
I mean dont use extra convertion to vertx.Json and vertx.Array